assets/src/blocks/Covers/TakeActionCovers.js

```diff
@@ -40,65 +36,61 @@ export const TakeActionCovers = ({ initialRowsLimit, covers, row, loadMoreCovers
           const buttonLink = inEditor ? null : button_link;
 
           return (
-            <div key={title} className='col-lg-4 col-md-6'>
-              <div className='cover-card'>
-                <a
-                  className='cover-card-overlay'
-                  data-ga-category='Take Action Covers'
-                  data-ga-action='Card'
-                  data-ga-label='n/a'
-                  href={buttonLink}
-                  aria-label={__('Take action cover, link to ' + title, 'planet4-blocks')}
+            <div key={title} className='col-lg-4 col-md-6 cover-card cover'>
+              <a
+                className='cover-card-overlay'
+                data-ga-category='Take Action Covers'
+                data-ga-action='Card'
+                data-ga-label='n/a'
+                href={buttonLink}
+                aria-label={__('Take action cover, link to ' + title, 'planet4-blocks')}
+              />
+              <a
+                data-ga-category='Take Action Covers'
+                data-ga-action='Image'
+                data-ga-label='n/a'
+                href={buttonLink}
+                aria-label={__('Take action cover, link to ' + title, 'planet4-blocks')}
+              >
+                <img
+                  alt={alt_text}
+                  src={image}
+                  srcSet={srcset}
+                  sizes={IMAGE_SIZES.takeAction}
                 />
+              </a>
+              <div className='cover-card-content'>
+                {/* Regardless of how many tags there are, we only show the first one */}
+                {tags && tags.length > 0 && <span className='cover-card-tag'>{tags[0].name}</span>}
```

Comment:
I think that `{tags && tags.length && ...` or maybe `{tags.length}` are fine too.

Reply:
I think we've already talked about this issue in another PR 😄 but if I just put `tags.length` then it will show "0" when there are no tags